src/main/java/service/ShopService.java

```diff
@@ -0,0 +1,41 @@
+package service;
+
+import model.Order;
+import model.Product;
+import repo.OrderRepo;
+import repo.ProductRepo;
+
+public class ShopService {
+
+    private ProductRepo productRepo;
+    private OrderRepo orderRepo = new OrderRepo();
+
+    public ShopService(ProductRepo productRepo) {
+        this.productRepo = productRepo;
+    }
+
+    public void addProduct(Product product){
+        productRepo.add(product);
+    }
+
+    public Product getProduct(String id){
+        return productRepo.get(id);
+    }
+
+    public void listProducts(){
+        productRepo.list();
```

Comment:
In den `list` und `get` Methoden des Services rufst du die richtigen Methoden auf - klasse!
Fehlt nur noch das du diese mittels `return` zurückgibst :)